saraho1123/overlook-sosgood
# Feature/guest interactions

## Is this a fix or a feature?
feature

## What is the change?
Add functionality for guest interactions:
As a customer:
* I should be able to select a date for which I’d like to book a room for myself
* Upon selecting a date, I should be shown a list of room details for only rooms that are available on that date
* I should be able to filter the list of available rooms by their roomType property
* I should be able to select a room for booking
* In the event that no rooms are available for the date/roomType selected, display a message fiercely apologizing to the user and asking them to adjust their room search

## What does it fix?
I fixed the filter bug!! Although that was a minor thing for this branch.

## Where should the reviewer start?
line 14 of Fetch.js
line 8 of Guest.js
line 1 of Hotel.js
line 57 of base.scss
line 15 of index.html
line 29 of index.js


## How should this be tested?
with repo cloned:
run: npm start in terminal
open: http://localhost:8080/ in browser
login using either manager or customer(# 01-50) and password of 'Overlook2020'.
Using any numbers <01 or >50 should cause an error message to appear below the submit button.
Once correctly logged in, all interactions for a guest should be working.
Note that there are several styling pieces and some dates that need to be fixed/completed.

## Relevant Tickets:
#17
